test/clojure-mode-font-lock-test.el

```diff
@@ -101,7 +101,13 @@ POS."
 
 (ert-deftest clojure-mode-syntax-table/fontify-clojure-keyword ()
   :tags '(fontification syntax-table)
-  (should (eq (clojure-test-face-at 2 11 "{:something 20}") 'clojure-keyword-face)))
+  (should (equal (clojure-test-face-at 2 11 "{:something 20}") '(clojure-keyword-face))))
+
+(ert-deftest clojure-mode-syntax-table/fontify-namespaced-keyword ()
+  :tags '(fontification syntax-table)
+  (should (equal (clojure-test-face-at 2 2 "{:alias/some 20}")  '(clojure-keyword-face)))
```

Comment:
> Guys, why?

For consistency with how other namespaced stuff is highlighted.

> It's very inconvenient now and it's weird.

This is just your opinion and it's very much influenced by having had namespaced keywords fontlocked in this manner for years.  If you'd never seen one before it would look perfectly reasonable.  In a few weeks time the new normal will be normal.

> It's so inconvenient that I'm looking now for a hack, how can I revert this behavior back without changing clojure-mode version...

You can copy in an old version of clojure-mode, which is to your liking, and either never update or cherry-pick changes you like.